acra/configuring-maintaining/logging/_index.md

```diff
@@ -1,4 +1,30 @@
 ---
-title: logging
+
+weight: 1
+title: Logging
 bookCollapseSection: true
 ---
+
+# Logging in Acra
+
+Starting with version [`0.77.0`](https://github.com/cossacklabs/acra/releases/tag/0.77.0), Acra supports three different logging formats: plaintext, [CEF](https://kc.mcafee.com/resources/sites/MCAFEE/content/live/CORP_KNOWLEDGEBASE/78000/KB78712/en_US/CEF_White_Paper_20100722.pdf), JSON.
+
+Acra logs are compatible with various external log analysing tools (like [ELK stack](https://www.elastic.co/elk-stack)) and SIEM systems.
```

Comment:
This would be a nice place to for a list:

```markdown
Starting with version [`0.77.0`](https://github.com/cossacklabs/acra/releases/tag/0.77.0),
Acra supports [multiple logging formats](formats/):

  - plaintext
  - CEF
  - JSON

Acra logs are compatible with various external log analysing tools (like [ELK stack](https://www.elastic.co/elk-stack)) and SIEM systems.
```